### PR TITLE
feat: add rabby wallet

### PR DESCRIPTION
**Pull request by heisenberg-2077**

### CODE REVIEW

File: packages/rainbowkit/src/wallets/getDefaultWallets.ts [hunk 1]
```diff
@@ -5,6 +5,7 @@ import { braveWallet } from './walletConnectors/braveWallet/braveWallet';
 import { coinbaseWallet } from './walletConnectors/coinbaseWallet/coinbaseWallet';
 import { injectedWallet } from './walletConnectors/injectedWallet/injectedWallet';
 import { metaMaskWallet } from './walletConnectors/metaMaskWallet/metaMaskWallet';
+import { rabbyWallet } from './walletConnectors/rabbyWallet/rabbyWallet';
```
Comment (DanielSinclair) [inline packages/rainbowkit/src/wallets/getDefaultWallets.ts:8]: Please revert these changes. We'll have a better solution soon for a community-driven list of wallets.

**Review by DanielSinclair — CHANGES_REQUESTED** [review]

**Comment by DanielSinclair** [comment]
@heisenberg-2077 Please add to `packages/example` for test dApp deployment and QA

**Comment by heisenberg-2077** [comment]
> @heisenberg-2077 Please add to `packages/example` for test dApp deployment and QA

I'm not sure what I have to modify, I can see the Rabby wallet in the test dApp.

File: packages/rainbowkit/src/wallets/getDefaultWallets.ts (hunk 1 above)
Comment (heisenberg-2077) [reply packages/rainbowkit/src/wallets/getDefaultWallets.ts:8]: What do I need to do now, I haven't seen a new solution yet.

Comment (DanielSinclair) [reply packages/rainbowkit/src/wallets/getDefaultWallets.ts:8]: @heisenberg-2077 Revert here and add to `packages/example` custom wallet list for testing

**Comment by heisenberg-2077** [comment]
Sorry I screwed up, i'll recreate a PR 😅
